challenge-4/submissions/evassilyev/solution-template.go

```
@@ -0,0 +1,108 @@
+        }
+}
+
+func bfs(graph map[int][]int, start int) []int {
+        visited := map[int]bool{}
+        queue := []int{start}
+        order := []int{}
+
+        visitedCounter := 0
+        for {
+
+                if len(graph) == visitedCounter {
```

golangci-lint (2.13.2)

[error] QF1006: could lift into loop condition

(staticcheck)